src/sentry/pipeline/__init__.py

```diff
@@ -232,6 +233,14 @@ def next_step(self):
         Render the next step.
         """
         self.state.step_index += 1
+        analytics.record(
```

Comment:
I'm wondering if this should be in `current_step` instead since [here](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/integration.py#L246) I'm not calling `next_step`, i'm manually setting the step index

Reply:
@MeredithAnya So it turns out that `current_step` gets a bunch of duplicate calls as you get further down the pipeline. What I did instead is add an optional argument for the `step_size` so we can skip ahead by more than one.